docs/config_bgp.md

```diff
@@ -0,0 +1,60 @@
+# Config BGP
```

Comment:
This file should be deleted, this might be getting pulled in from your local branch.

Reply:
this keeps getting regenerated because:
```
$ ls docs/_build/markdown/config_bgp.md 
docs/_build/markdown/config_bgp.md
ls docs/_build/doctrees/config_bgp.doctree 
docs/_build/doctrees/config_bgp.doctree
```

will remove all three files

Reply:
`canu/config/bgp` directory still exists.